tests/e2e/driver/CheReporter.ts

```diff
@@ -136,8 +138,12 @@ class CheReporter extends mocha.reporters.Spec {
       // raise flag for keeping the screencast
       deleteScreencast = false;
 
-      const testFullTitle: string = test.fullTitle().replace(/\s/g, '_');
-      const testTitle: string = test.title.replace(/\s/g, '_');
+      Logger.trace(`FullTitle:${test.fullTitle()}`);
+      const testFullTitle: string = sanitizer.sanitize(test.fullTitle());
+      Logger.trace(`FullTitleSanitized:${testFullTitle}`);
+      Logger.trace(`TestTitle:${test.title}`);
+      const testTitle: string = sanitizer.sanitize(test.title);
+      Logger.trace(`TestTitleSanitized:${testTitle}`);
```

Comment:
I know it's trace.... But wouldn't it be enough to have it printed once?
Or maybe even remove the log from here. This method is being called pretty often and I'm afraid it will overwhelm the log output.